jetty-websocket/javax-websocket-client/src/main/java/org/eclipse/jetty/websocket/javax/client/JavaxWebSocketClientContainer.java

```diff
@@ -114,6 +110,8 @@ protected WebSocketCoreClient getWebSocketCoreClient()
                 addBean(coreClient,false);
             else
                 addManaged(coreClient);
+
+            coreClient.setCustomizer(defaultCustomizer);
```

Comment:
Hmmm sorry but I still don't like this.
If the coreClient can be running, then you are saying it can be shared, then each client container is going to fight over setting the customizer.
Can it be shared now?  probably not, so perhaps remove the code above.....

But still I would prefer that we supported shareable components, so I still don't like setting our customizer here.  It would be better to apply our customizer to session created in connect below.   @sbordet what do you think about being able to share the coreClient?

Reply:
ahh yes i forgot about this, now that we are supplying the defaultCustomizer from the Container in the CoreClient constructor we do not need the getter and setter for the ConfigurationCustomizer on the CoreClient

so this code can be removed, i will fix it now

Reply:
let's chat to @sbordet first about if we should allow for a shared coreClient....